modules/tinymce/src/core/main/ts/fmt/MatchFormat.ts

```diff
@@ -8,13 +8,14 @@
 import { Arr } from '@ephox/katamari';
 import DOMUtils from '../api/dom/DOMUtils';
 import Editor from '../api/Editor';
-import { Format, FormatVars, SelectorFormat } from '../api/fmt/Format';
+import { Format, FormatVars, SelectorFormat } from './FormatTypes';
 import * as FormatUtils from './FormatUtils';
 
 const isEq = FormatUtils.isEq;
 
 const matchesUnInheritedFormatSelector = function (ed: Editor, node: Node, name: string) {
-  const formatList = ed.formatter.get(name);
+  // TODO: Is this safe? it doesn't look like it is this could be a block or inline format
+  const formatList = ed.formatter.get(name) as SelectorFormat[];
```

Comment:
Would it be worth adding a `FormatUtils.isSelectorFormat(format)` check to the below `if` statement instead?

Reply:
I considered that, but I don't know for sure if `inherit` only applies to selector formats. So I went with the safer option and left the TODO for now.